website/templates/project/settings.mako

```diff
@@ -474,6 +497,7 @@
       window.contextVars.wiki.isEnabled = ${wiki.short_name in addons_enabled | sjson, n };
       window.contextVars.currentUser = window.contextVars.currentUser || {};
       window.contextVars.currentUser.institutions = ${ user['institutions'] | sjson, n };
+      window.contextVars.user = ${ user | sjson, n };
```

Comment:
I was wrong, ctx.currentUser has less info than ctx.user. In our case, currentUser does not have permissions info, which sharing-page needs.

ctx.currentUser:
https://github.com/CenterForOpenScience/osf.io/blob/develop/website/templates/project/project_base.mako#L68

ctx.user:
https://github.com/CenterForOpenScience/osf.io/blob/develop/website/project/views/node.py#L679